spec/javascripts/unit/admin/line_items/controllers/line_items_controller_spec.js.coffee

```diff
@@ -261,24 +261,24 @@ describe "LineItemsCtrl", ->
           scope.formattedValueWithUnitName(1,unitsProduct,unitsVariant)
           expect(Math.round).toHaveBeenCalled()
 
-        it "calls Math.round with the quotient of scale and value, multiplied by 1000", ->
+        it "calls Math.round with the value multiplied by 1000", ->
```

Comment:
It's good practice to add spec changes to the same commit as the code changes that require them. So you don't have commits with broken specs and you can see why the spec changes are needed.

Reply:
I totally agree, and I usually try to do it as I can. This time, it was wild (as you said) and I missed to link commits to spec changes.